layouts/partials/head/favicons.html

```diff
@@ -1,5 +1,6 @@
 <meta name="theme-color" content="{{ $.Site.Params.themeColor }}">
-<link rel="apple-touch-icon" sizes="180x180" href="{{ "apple-touch-icon.png" | relURL }}">
-<link rel="icon" type="image/png" sizes="32x32" href="{{ "favicon-32x32.png" | relURL }}">
-<link rel="icon" type="image/png" sizes="16x16" href="{{ "favicon-16x16.png" | relURL }}">
+<!--link rel="apple-touch-icon" sizes="180x180" href="{{ "apple-touch-icon.png" | relURL }}"-->
+<link rel="icon" type="image/png" sizes="49x49" href="{{ "DataTrails_Favicon.png" | relURL }}">
```

Comment:
This is really supposed to be 49, not 48?

Reply:
It was the size that I downloaded from sharepoint. I'll change it to 48